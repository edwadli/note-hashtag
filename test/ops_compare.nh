if -42.0 < 42.0 then PrintEndline "-42.0 < 42.0" else PrintEndline "failed L1"
if 1000 > 0 then PrintEndline "1000 > 0" else PrintEndline "failed L2"
if 1 <= 1 then PrintEndline "1 <= 1" else PrintEndline "failed L3"
if 1 <= 2 then PrintEndline "1 <= 2" else PrintEndline "failed L4"
if -9 >= -9 then PrintEndline "-9 > -9" else PrintEndline "failed L5"
if 2.8 >= -9e+3 then PrintEndline "2.8 >= -9e+3" else PrintEndline "failed 6"
