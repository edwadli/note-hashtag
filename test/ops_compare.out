-42.0 < 42.0
1000 > 0
1 <= 1
1 <= 2
-9 > -9
2.8 >= -9e+3
